docs/reference/cluster/pending.asciidoc

```diff
@@ -47,3 +48,5 @@ like this:
   ]
 }
 --------------------------------------------------
+// NOTCONSOLE
```

Comment:
I'd originally thought of `// NOTCONSOLE` as only for requests that looked like they should be converted to `// CONSOLE` but can't for some reason, like maybe we're demonstrating how to use curl in the `_bulk` API.

While the name `// NOTCONSOLE` isn't really right for this the idea makes sense. We can't assert anything about this response.

So I'm :+1: on doing this here.